cmd/kyverno/main.go

```diff
@@ -470,6 +473,16 @@ func main() {
 	kubeInformer := kubeinformers.NewSharedInformerFactory(kubeClient, resyncPeriod)
 	kubeKyvernoInformer := kubeinformers.NewSharedInformerFactoryWithOptions(kubeClient, resyncPeriod, kubeinformers.WithNamespace(config.KyvernoNamespace()))
 	kyvernoInformer := kyvernoinformer.NewSharedInformerFactory(kyvernoClient, resyncPeriod)
+	// this label need to be hard-coded as at this point we don't have any resource to get this value from
+	labelsMap := map[string]string{"cache.kyverno.io/enabled": "true"}
+	labelSelector := labels.Set(labelsMap).AsSelector().String()
+	kubeResourceInformer := kubeinformers.NewFilteredSharedInformerFactory(
```

Comment:
It's simple to use `NewSharedInformerFactoryWithOptions`

Reply:
fixed